RELEASING.md

```diff
@@ -100,29 +136,43 @@ Please note:  This process is for regularly scheduled minor and patch releases.
 * Release ticket is cut, and there's a forum post announcing the start of the window.
 * [Any necessary security reviews](##Security-Reviews) are complete. 
 
-#### Exit Criteria to Close Release Window
+##### Exit Criteria to Close Release Window
+
 * Performance tests are run, results are posted to the release ticket and there no unexpected regressions
 * Documentation has been fully reviewed and signed off by the documentation community.
 * All integration tests are passing
 * Release blog is ready
 
-### OpenSearch Major Releases
+### Other Projects
+
+Look for a RELEASING.md file in each repo, e.g. [opensearch-java/RELEASING.md](https://github.com/opensearch-project/opensearch-java/blob/main/RELEASING.md). This is typically a [1-click release process](https://github.com/opensearch-project/opensearch-build/blob/main/ONBOARDING.md#onboarding-to-universal--1-click-release-process) in which a maintainer pushes a tag to the repo, followed by automation performing the actual release.
+
+### Tagging
 
-OpenSearch only does major releases when there are significant breaking changes that are ready for release.  Once we become aware of the need for a major version, we will start a major version release window which will be similar to a minor release, except for two things:  Participation is mandatory for all components and the release window will be at least 4 weeks long to accommodate the testing required. 
+Create tags after a release that match the version number, `major.minor.patch`, without a `v` prefix.
```

Comment:
could we elaborate why we choose to tag without `v`, it is very confusion compare to branch. large oss project like linux, react use v- for tag

https://github.com/torvalds/linux/tags
https://github.com/facebook/react/tags

Reply:
Below it says

_For a discussion on whether to add a prefixing `v` to release tags, see [#35](https://github.com/opensearch-project/.github/issues/35)._

Reply:
Thanks, @dblock. It doesn't explain why we chose not to add the "v-" at the beginning, but I understand it was a concern. there are effort to support, but no one work on it. and the world have gotten used to it now. 

Good to know, but we should try in new repository. Especially for those repository we took the ownership for maintenance from other open source org. like fork new repo.